dev-environment/h2o/models/glm-wine/train.py

```diff
@@ -3,29 +3,33 @@
 import os
 from h2o.estimators.glm import H2OGeneralizedLinearEstimator
 
-if __name__ == "__main__":
-    h2o_port = int(os.environ['H2O_SERVER_PORT'])
-    h2o.init(port = h2o_port)
+if __name__ != "__main__":
+    sys.exit()
 
-    csv_url = ("http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv")
-    try:
-        data = h2o.import_file(csv_url)
-    except Exception as e:
-        logger.exception("Unable to download training & test CSV, check your internet connection. Error: {}".format(e))
+print(sys.argv)
+alpha = float(sys.argv[1])
```

Comment:
Are we actually using this alpha anywhere?

Reply:
now we use, it was missed, thanks